src/engine/source/kvdb/src/kvdb.cpp

```diff
@@ -78,16 +78,14 @@ struct KVDB::Impl
         auto s = rocksdb::DB::ListColumnFamilies(kOptions.open, m_path, &cfNames);
         if (s.ok())
         {
-            for (auto name : cfNames)
+            for (const auto& name : cfNames)
             {
-                m_CFDescriptors.push_back(
-                    rocksdb::ColumnFamilyDescriptor(name, kOptions.CF));
+                m_CFDescriptors.emplace_back(name, kOptions.CF);
             }
         }
         else
         {
-            m_CFDescriptors.push_back(
-                rocksdb::ColumnFamilyDescriptor(DEFAULT_CF_NAME, kOptions.CF));
+            m_CFDescriptors.emplace_back(DEFAULT_CF_NAME, kOptions.CF);
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-(C-plus-plus)#modern-language-features-and-best-practices

Use emplace_back instead of push_back

Reply:
Just a question: why are we removing `rocksdb::ColumnFamilyDescriptor()` ?

Reply:
becasue I use emplace_back, the allocation is implicit.

Reply:
The question was answered, thank you.